ui/tslint.json

```diff
@@ -80,6 +86,7 @@
     ],
     "radix": true,
     "semicolon": [
+      true,
```

Comment:
Sounds good - please create a follow up issue as it would affect a large number of files.